Spec/Tests/RestClientChannelTests.swift

```diff
@@ -1488,4 +1488,47 @@ class RestClientChannelTests: XCTestCase {
             }
         }
     }
+    
+    // RSL8a, CHD2b, CHS2b, CHO2a
+    func test__047__channel_with_subscribers_status_method_returns_a_channel_details_object_with_fullfilled_channel_metrics() {
+        let options = AblyTests.commonAppSetup()
+        let rest = ARTRest(options: options)
+        let realtime = ARTRealtime(options: options)
+
+        let channelName = uniqueChannelName()
+        
+        let realtimeChannel = realtime.channels.get(channelName)
```

Comment:
In addition to the new descriptions, turned out that to increase _all_ `metrics` by 1 it's enough to call `enter:` in `presence` in realtime channel - https://github.com/ably/ably-cocoa/pull/1420/commits/a1b14095b3822be654a08d04818e5eeaa8f30c40

Reply:
@SimonWoolf Couldn't you please reassure that this is expected behavior for all metrics in this case? Also I would appreciate if you suggest a proper test name for it, f.e. `test__047__channel_with_presence_members_status_method_returns_a_channel_details_object_with_fullfilled_channel_metrics`, does it accurately reflect what's going on here? Thanks

Reply:
> please reassure that this is expected behavior for all metrics in this case?

yes

tbh I'm not sure I'd worry about all of this that much? In the sense that: if the client library is successfully obtaining the occupancy object, it has done all that it can or should do; making sure that the returned object is _accurate_ is the server's responsibility

Reply:
> tbh I'm not sure I'd worry about all of this that much? In the sense that: if the client library is successfully obtaining the occupancy object, it has done all that it can or should do; making sure that the returned object is accurate is the server's responsibility

Agreed, but since our general approach to tests in this SDK has been one that uses very little mocking, we don't really have a way to verify that the SDK is successfully parsing the object contained in the response other than by creating a scenario that provokes a certain expected response from the server and then asserting the object returned by the SDK matches that expected response. It's probably not ideal.